release-content/migration-guides/rename-justifytext.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Renamed `JustifyText` to `Justify`
+pull_requests: [19522]
+---
+
+`JustifyText` has been renamed to `Justify`.
+
+The `-Text` suffix was inconsistant with the names of other `bevy_text` types and unneccessary since it's natural to assume `Justify` refers to text justification.
```

Comment:
```suggestion
The `-Text` suffix was inconsistent with the names of other `bevy_text` types and unnecessary since it's natural to assume `Justify` refers to text justification.
```